### PR TITLE
Add section "Image scales from brain" to Classic UI > Image handling

### PR DESCRIPTION
**Pull request by MrTango**

### CODE REVIEW

File: docs/classic-ui/images.md [hunk 1]
```diff
@@ -460,3 +460,51 @@ This will result in a `srcset` as in the following example for a medium image:
 Please note that this example has the `resolve_uid_and_caption` filter disabled to see the scale names better.
 The real `src` URLs look more like `http://localhost:8080/Plone50/dsc04791.jpg/@@images/778f9c06-36b0-485d-ab80-12c623dc4bc3.jpeg`.
 ```
+
+## Image scales from catalog brain
+
+For all `NamedBlobImage` fields, we can get existing scale info's directly from the catalog brain.
```
Comment (stevepiercy) [inline docs/classic-ui/images.md]:
```suggestion
For all `NamedBlobImage` fields, we can get existing scale information directly from the catalog brain.
```

File: docs/classic-ui/images.md [hunk 2]
```diff
@@ -460,3 +460,51 @@ This will result in a `srcset` as in the following example for a medium image:
...
+
+Given a content type with a `NamedBlobField` named `picture`, we can get the following info's, by calling the `image_scales` attribute on the catalog brain.
```
Comment (stevepiercy) [inline docs/classic-ui/images.md]:
```suggestion
Given a content type with a `NamedBlobField` named `picture`, we can get the following information by calling the `image_scales` attribute on the catalog brain.
```

File: docs/classic-ui/images.md [hunk 3]
```diff
@@ -460,3 +460,51 @@ This will result in a `srcset` as in the following example for a medium image:
...
+                                   'height': 128,
+                                   'width': 128},
+                         'tile': {'download': '@@images/picture-64-220d6703eac104c59774a379a8276e76.jpeg',
+                                  'height': 64,
+                                  'width': 64}},
+              'size': 238977,
+              'width': 800}]}
+```
+
+In these info's we have everything we need to generate our image URL's, without waking up any object's.
```
Comment (stevepiercy) [inline docs/classic-ui/images.md]:
```suggestion
This information shows we have everything we need to generate our image URLs, without waking up any objects.
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Nice! Just a few grammar fixes, and I approve.

**Review by petschki — DISMISSED** [review]
Nice!

**Review by stevepiercy — APPROVED** [review]

**Comment by stevepiercy** [comment]
I'm going ahead and merging this PR, as the failed test is for linkcheck and webarchive.org has once again fallen down.

**Pull request merged**